From 8d01f1419c62e24b662abc2e1ec708a7c63fbbfe Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Wed, 1 Jul 2020 17:00:02 -0400
Subject: [PATCH] Use +self_spec: instead of *self_spec:

After discussion with NIIBE Yutaka on https://dev.gnupg.org/T4982 and
Guillem Jover on IRC, I think this is the correct fix for problems
when compiling Qt/GPGME code in debian systems.

I don't fully understand the implications of this change, but i
believe it is related to #870383 and #879014 (in the debian BTS) as
well.
---
 data/no-pie-compile.specs | 2 +-
 data/no-pie-link.specs    | 2 +-
 data/pie-compile.specs    | 2 +-
 data/pie-link.specs       | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/data/no-pie-compile.specs b/data/no-pie-compile.specs
index 2277b97ef..70cb36095 100644
--- a/data/no-pie-compile.specs
+++ b/data/no-pie-compile.specs
@@ -1,2 +1,2 @@
-*self_spec:
++self_spec:
 + %{!r:%{!fpie:%{!fPIE:%{!fpic:%{!fPIC:%{!fno-pic:-fno-PIE}}}}}}
diff --git a/data/no-pie-link.specs b/data/no-pie-link.specs
index 54db649b1..fa4162793 100644
--- a/data/no-pie-link.specs
+++ b/data/no-pie-link.specs
@@ -1,2 +1,2 @@
-*self_spec:
++self_spec:
 + %{!shared:%{!r:%{!fPIE:%{!pie:-fno-PIE -no-pie}}}}
diff --git a/data/pie-compile.specs b/data/pie-compile.specs
index 74d82155c..c1ee08c71 100644
--- a/data/pie-compile.specs
+++ b/data/pie-compile.specs
@@ -1,2 +1,2 @@
-*self_spec:
++self_spec:
 + %{!r:%{!fpie:%{!fPIE:%{!fpic:%{!fPIC:%{!fno-pic:%{!fno-PIE:%{!no-pie:-fPIE}}}}}}}}
diff --git a/data/pie-link.specs b/data/pie-link.specs
index 94c122fd3..9b401e34a 100644
--- a/data/pie-link.specs
+++ b/data/pie-link.specs
@@ -1,2 +1,2 @@
-*self_spec:
++self_spec:
 + %{!static:%{!shared:%{!r:%{!fno-PIE:%{!no-pie:-fPIE -pie}}}}}
-- 
2.27.0

